### ISSUE TITLE
Popover's body/content removed after hiding all popovers with 'bv::hide::popover' on $root

### ISSUE THREAD

**Issue by josespinal** [bootstrap-vue/bootstrap-vue#1322]
Hi, 

When I try to close all open popovers on the "@show" event:

```
onShow () {
      this.$root.$emit('bv::hide::popover')
}
```
And I try to reopen the popover, I get empty content on it. What could it be?

Edit 1: What gets deleted is the inline content inside the component.

Thanks in advance!!

**Comment by tmorehouse** [comment]
I assume you are using the component version of popover?

Could you provide some sample markup & code?

**Comment by tmorehouse** [comment]
We were able to recreate the issue, and have a PR that will fix this.

Should be available in the next release.

**Comment by josespinal** [comment]
Perfect! Thanks!

**Comment by josespinal** [comment]
Made the changes on the commit, but the problem persists. Anything else needed?

**Comment by tmorehouse** [comment]
I can't reproduce anymore with the latest dev branch which has the commit.

**Comment by tmorehouse** [comment]
I used the following markup as a test:
```html
  <div>
    <b-button id="a">button</b-button>
    <b-button id="b">button</b-button>
    <b-button id="c">button</b-button>
    <b-button id="d">hide others</b-button><br><br>
    <b-popover target="a" triggers="click" placement="top">Hello</b-popover>
    <b-popover target="b" triggers="click" placement="top">Hello</b-popover>
    <b-popover target="c" triggers="click" placement="top">Hello</b-popover>
    <b-popover target="d" triggers="click" placement="top" @show="onShow">Hello</b-popover> 
  </div>
```

and this script:
```js
{
    methods: {
      onShow() {
        this.$root.$emit('bv::hide::popover')
      }
    }
}
```

The docs site is running the latest dev branch, so you can past the above in the playground to see it in action.

**Comment by tmorehouse** [comment]
Fixed in release v1.1.0
